client/vscode/src/backend/requestGraphQl.ts

```diff
@@ -15,7 +15,8 @@ export function invalidateClient(): void {
 
 export const requestGraphQLFromVSCode = async <R, V = object>(
     request: string,
-    variables: V
+    variables: V,
+    overrideAccessToken?: string
```

Comment:
Should this be a default param instead? I was a bit confused why we need to have two different access token concepts but I think the reason is that we want to issue requests as a different access token to validate a new user input before persisting it right?

```suggestion
   accessToken: string = accessTokenSetting()
```

Reply:
I ended up keeping it so we know when to suppress the invalid access token notification: https://github.com/sourcegraph/sourcegraph/blob/b998306da163b0c59c4db25fecefbfa7f3d164cb/client/vscode/src/backend/requestGraphQl.ts#L60-L64